components/_buttons/buttons.scss

```diff
@@ -21,18 +21,19 @@
 @mixin dark-theme-disabled {
   background-color: $rgba-11;
   border: 1px solid $rgba-8;
-  color: $gray-7;
+  color: rgba(255,255,255,.5);
```

Comment:
Check out _variables.scss, we have variables for these rgba colours

$rgba-7: rgba(255, 255, 255, 0.5);